rally-domain/src/test/groovy/yakworks/rally/orgs/ContactRepoSpec.groovy

```diff
@@ -38,6 +38,13 @@ class ContactRepoSpec extends Specification implements DataRepoTest, SecurityTes
         then:
         c
         c.id == contact.id
+
+        when: "Lookup from sources collection"
+        c = contactRepo.lookup(sources:[[sourceId:'123']])
```

Comment:
the test here will not be `contactRepo.lookup(sources:[[sourceId:'123']])` it will be `contactRepo.lookup(sourceId:'123')`, again see OrgRepo, that is the loose model for this